include/variant_detection/variant_detection.hpp

```diff
@@ -10,6 +11,7 @@
  *         detected junctions are stored in a vector.
  *
  * \param[in, out]  junctions - a vector of junctions
+ * \param[in, out]  sn_ln_tag_tuple_set - a vector of @SQ header tags tuples: SN and LN
```

Comment:
I'm not sure about the name `sn_ln_tag_tuple_set` any better ideas?

Reply:
Maybe just `sn_ln_tags`?
My thinking is that there is no need to encode the data type into the variable name (`set` and `tuple`).

That it is a `set` and `tuples` is not so relevant and also clear from the datatype.

If I see it correctly, SN means "sequence number" and LN is the length?
So maybe `seq_length` or `sequence_length`. (I don't like this one so much, also it seems like it is a `map` not a `set`).